pulp_python/app/serializers.py

```diff
@@ -299,17 +338,19 @@ def create(self, validated_data):
 
         """
 
-        projects = validated_data.pop('projects')
+        projects = validated_data.pop("projects")
 
         python_remote = python_models.PythonRemote.objects.create(**validated_data)
         for project in projects:
-            digests = project.pop('digests', None)
-            specifier = python_models.ProjectSpecifier.objects.create(remote=python_remote,
-                                                                      **project)
+            digests = project.pop("digests", None)
+            specifier = python_models.ProjectSpecifier.objects.create(
+                remote=python_remote, **project
+            )
             if digests:
                 for digest in digests:
-                    python_models.DistributionDigest.objects.create(project_specifier=specifier,
-                                                                    **digest)
+                    python_models.DistributionDigest.objects.create(
+                        project_specifier=specifier, **digest
+                    )
```

Comment:
I'm not sure I like how it does this.  It makes sense in a bunch of places but it feels wrong here.